styles.css

```diff
@@ -0,0 +1,91 @@
+h2 {
+  font-family: "Playfair Display SC", serif;
+  font-size: 42px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+
+  line-height: 5px;
```

Comment:
Here, you are taking the `line-height` way way way down -- but what you actually want to control/override is the default margins on your heading.